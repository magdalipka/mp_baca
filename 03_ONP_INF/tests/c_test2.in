19
ONP: xabc**=
ONP: ab+a~a-+
ONP: ( a,b,.).c;-,*
ONP: abc++def++g+++
ONP: xabc=== 
ONP: abcd^e-fgh*+^*+i- 
ONP: +a+b
ONP: a~~
ONP: abc-+de^/
ONP: -abc-+de^/
ONP: a
ONP: ab*cd*+de*+fg*+
ONP: abcd+++
ONP: ab+c+d+
ONP: ab*cd**
ONP: ab/c*
ONP: ab+cd++
ONP: ab+cd*/
ONP: abc-+de-fg-h+/*