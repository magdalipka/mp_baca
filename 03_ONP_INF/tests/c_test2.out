INF: x = a * ( b * c )
INF: a + b + ( ~ a - a )
INF: a * ( b - c )
INF: error
INF: x = a = b = c
INF: a + b * ( c ^ d - e ) ^ ( f + g * h ) - i
INF: error
INF: ~ ~ a
INF: ( a + ( b - c ) ) / d ^ e
INF: error
INF: a
INF: a * b + c * d + d * e + f * g
INF: a + ( b + ( c + d ) )
INF: a + b + c + d
INF: a * b * ( c * d )
INF: a / b * c
INF: a + b + ( c + d )
INF: ( a + b ) / ( c * d )
INF: ( a + ( b - c ) ) * ( ( d - e ) / ( f - g + h ) )